docs/operate-and-deploy/migrations-tool.md

```diff
@@ -48,6 +48,8 @@ types of ksqlDB statements:
 * `DROP TYPE`
 * `SET <property>`
 * `UNSET <property>`
+* `DEFINE <variable>`
```

Comment:
Let's clarify the scope for these different commands. For example, `DEFINE` and `UNDEFINE` apply for the specific migrations file but not beyond that. It'd be good to clarify the scope for `SET` and `UNSET` too.